src/services/followMainFeedFollowing.js

```diff
@@ -1,23 +1,25 @@
-const stream = require("getstream");
+const stream = require('getstream');
+const UserService = require('../services/postgres/UserService');
 
 const followMainFeedFollowing = async (userId, userIds) => {
   if (!userIds || userIds.length == 0) {
     return;
   }
 
   const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
+  const userSevice = new UserService();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The connection to the stream and the creation of a new UserService instance should be handled outside of this function. This will improve performance by reusing the same connection and instance across multiple function calls, rather than creating a new one each time this function is called.
```diff
- const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
- const userSevice = new UserService();
+ let cs;
+ let userService;
+ 
+ if (!cs) {
+   cs = stream.connect(process.env.API_KEY, process.env.SECRET);
+ }
+ 
+ if (!userService) {
+   userService = new UserService();
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->